CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/application-preferences.md

```diff
@@ -62,3 +62,13 @@ location accessed for the next session. This can be altered in
 preferences by unticking the "Remember Open/Save Location" option. When
 this is unticked, the open or save location defaults to the user's home
 directory.
+
+## Application Font Preferences
+
+Application Font preferences can be accessed via Setup -> Options ->
+CONSTELLATION -> Application.
```

Comment:
I think this was just the opening to the old help page so this could probably be omitted